itunes-link-engine/trunk/options.php

```diff
@@ -765,10 +765,36 @@ function getGeoriotAffiliates(suppressError) {
         </div>
       </div>
     </div>
+	<div id="gr-advanced-options" class="gr-step-area">
+      <h5>
+        <em class="gr-expand">+</em>
+        <em class="gr-collapse">--</em>
+        Advanced Options<span class="hidden-expanded">...</span>
+      </h5>
+      <div class="gr-advanced-options-fields">
+        <br>
+        <div class="gr-tsid-spinner">
+          <div class="css-only-spinner">
+            <div class="bounce1"></div>
+            <div class="bounce2"></div>
+            <div class="bounce3"></div>
+          </div>
+          Loading your groups...
+        </div>
+        <br>
+		<input type="checkbox" name="genius_ile_urls_on_click" value="yes"
+            <?php if (get_option('genius_ile_urls_on_click') == 'yes') print "checked" ?> />Preserve URLs until click time
+        <a href="#ile-faq-on-click">(?)</a>
+		<br><br>
+		<input type="checkbox" name="genius_ile_api_remind" value="yes"
+			<?php if (get_option('genius_ile_api_remind') == 'yes') print "checked" ?> />
+		Show Wordpress alert on dashboard if commissions are not enabled
```

Comment:
That looks like a comment for yourself

Reply:
I think this is supposed to be there. It's in the mock-up.

Reply:
ahh got it.

Reply:
@ssundheim that text in there was just an instruction to the developer right ?

Reply:
@aallfredo we have a user option in the plugin to "Show Wordpress alert on dashboard if commissions are not enabled". It's on by default, so users know they still need to add their api keys.  This option is basically just for people who are ok with not getting commissions and want the alert to go away.

Reply:
thanks!